soda/core/soda/execution/check/schema_check.py

```diff
@@ -191,33 +190,123 @@ def has_schema_violations(
+                        "details": f"Previous Index: {index_change['previous_index']}; New Index: {index_change['new_index']}",
+                    }
+                )
+            for column, type_change in self.schema_comparator.schema_column_type_changes.items():
+                change_events.append(
+                    {
+                        "column": column,
+                        "event": "Type Changed",
+                        "details": f"Previous Type: {type_change['previous_type']}; New Type: {type_change['new_type']}",
+                    }
+                )
+        else:
+            # Create diagnostics blocks without previous schema comparison.
+            # Simply take outcome type and count of all changes. Current logic bails out on fail condition so warn events are not present if fail is.
+            for column, index_mismatch in self.schema_column_index_mismatches.items():
+                change_events.append(
+                    {
+                        "column": column,
+                        "event": "Index Mismatch",
+                        "details": f"Expected Index: {index_mismatch['expected_index']}; Actual Index: {index_mismatch['actual_index']}",
+                    }
+                )
+            for column, type_mismatch in self.schema_column_type_mismatches.items():
+                change_events.append(
+                    {
+                        "column": column,
+                        "event": "Type Mismatch",
+                        "details": f"Expected Type: {type_mismatch['expected_type']}; Actual Type: {type_mismatch['actual_type']}",
+                    }
+                )
+            for column in self.schema_present_column_names:
+                change_events.append(
+                    {
+                        "column": column,
+                        "event": "Forbidden Column Present",
+                        "details": "",
+                    }
+                )
+            for column in self.schema_missing_column_names:
+                change_events.append(
+                    {
+                        "column": column,
+                        "event": "Required Column Missing",
+                        "details": "",
+                    }
+                )
+        event_count = len(change_events)
+
+        if change_events:
+            change_events_text = "\n".join(
+                f"{ch['column']},:icon-{value_type}: {ch['event']}, {ch['details']}" for ch in change_events
+            )
+            schema_diagnostics["blocks"].append(
+                {
+                    "type": "csv",
+                    "text": f"Column,Event,Details\n{change_events_text}",
+                    "title": "Diagnostics",
+                }
+            )
+
+        schema_diagnostics["preferredChart"] = "bars"
+        schema_diagnostics["valueLabel"] = f"{event_count} schema event(s)"
+        schema_diagnostics["valueSeries"] = {
```

Comment:
Warning: we are limiting `valueSeries` to 10 items max per test result

Reply:
removing schema checks from this PR, this will be dealt with separately